### ISSUE TITLE
Credits

### ISSUE THREAD

**Issue by SuatEyrice** [HaxeExtension/extension-webview#13]
You have copied my codes from repository and changed some namespace and not giving any credits.

https://github.com/openfl/extension-webview/blob/master/project/ios/WebViewEx.mm
https://github.com/SuatEyrice/NMEWebview/blob/master/extensions/WebView/project/iphone/NMEWebView.mm

**Comment by SuatEyrice** [comment]
Also most of this code too.

https://github.com/SuatEyrice/NMEWebview/blob/master/src/com/arcademonk/webview/WebView.hx
https://github.com/openfl/extension-webview/blob/master/extension/webview/WebView.hx

**Comment by fbricker** [comment]
Hi, we've made this extension together with @matiasjrossi and @DanielUranga time ago using one open-source extension we've found on github as the base for the iOS implementation (I guess it was yours, but it was couple of years ago).

Initially it wasn't our idea to release the extension (was just for internal use so we didn't even had any documentation or place were to put credits), later on we've decided to release it, but we forgot to add the original extension on the credits (in fact we've forgotten which was the original extension).

So, sorry for not placing you on the credits. I'll fix this now. It's not that we don't like to give credits: We always do that if we take some else's code as the base for something... (check https://www.muses.org/open-source, https://github.com/SempaiGames/haxe-ga, or https://github.com/SempaiGames/haxe-gif to verify we actually do put people on the credits).
... in this case we just forgot :(

Anyway, thanks for publishing your code in the first place :) I'll be adding you
